src/Components/ServiceScene/Breakdowns/SortByScene.tsx

```diff
@@ -28,6 +28,11 @@ export class SortByScene extends SceneObjectBase<SortBySceneState> {
           label: 'Most relevant',
           description: 'Smart ordering of graphs based on the most significant spikes in the data',
         },
+        {
+          value: 'outliers',
+          label: 'Detected outliers',
+          description: 'Order by detected outliers in the data',
```

Comment:
What does that mean, btw? Is it more "Order by the count/amount of outliers"?

Reply:
It's a good question. Let's add more details to the description.